### ISSUE TITLE
Error when running script on Windows 'no such file or directory'

### ISSUE THREAD

**Issue by Croman12** [caronc/nzb-notify#87]
:question: **Question**

I'm trying to set it up on my Windows pc, but I seem to have problems getting the script to run.

I installed the script in the directory (the .py file and the Notify folder), when I put in my servers, I get this error message.
For sure it must be easy to fix, but I have zero experience with Python and have no idea where to start.

I found on the internet that maybe the directories for Python are different, but I think I can see in the log that it is not the problem. I changed the NZBGet server running as System and running as my personal account, but that didn't seem to work.
When I run the .py file in IDLE, I get the "nzbget module missing', that may be because I'm running it in NZBGet (or is this also not intended?).

This is the error message:

<Executing script Notify.py with command TestServers Notify: C:\Program Files\Python39\python.exe: can't open file 'C:\Windows\system32\L': [Errno 2] No such file or directory Script Notify.py with command TestServers failed (terminated with unknown status)
>

Any help is greatly appreciated.

**Comment by Croman12** [comment]
Okay so a lot of progress since yesterday. When I manually run the file with the parameter for telegram, I receive a test message on Telegram. There was an error though, and I'm not sure what it means: 

`Warning (from warnings module):
  File "C:\Users\NAME\AppData\Roaming\Python\Python39\site-packages\apprise\utils.py", line 704
    pmatch = re.search(
FutureWarning: Possible nested set at position 12
2022-04-27 00:19:32,082 - 2084 - INFO - Notifying 1 service(s) asynchronously.
2022-04-27 00:19:32,246 - 2084 - INFO - Sent Telegram notification.
>>> `

In NZBGet I still cannot run is and I get the same error message as before, it has to be something with the quotation marks or with how NZBGet is running the script. I run NZBGet as a service as my local Administrator account.

Again, any help is greatly appreciated, especially since I feel I'm really close now.

**Comment by caronc** [comment]
Seems like you may not have the `Notify.py` script in the correct directory.  Also reload NZBGet so it can see it (once you've placed it).

You will be able to run tests right from in NZBGet once this is all in place.

If you can run from the command line then you're halfway there for sure. NZBGet needs to be able to access your `python.exe` file on your PC. Make sure it's in the path and/or configured in NZBGet.

**Comment by Croman12** [comment]
I finally did it. I don't know why, because everything in environment is not directing at all to the system32 folder, but apparently something is still looking for it. I put ".py=C:\Python310\python.exe"  in shell override option in nzbget, and everything is working now. Thank you for your help and response!

One quick question, when a download finishes, all the script sends is a picture or should there also be some text in the message?

**Comment by caronc** [comment]
Yes, you should receive text as well. What notification service are you using? In NZBGet you can also send a test notification to make sure it's all hooked up properly.

**Comment by Croman12** [comment]
I use Telegram. When a download starts, I get a notification saying "New file queued for donload <NZB>". But when the download is finished, I only get the succes image (apprise-success-128x128.png).

**Comment by caronc** [comment]
I did a mini-release of Apprise to fix this [here](https://github.com/caronc/apprise/pull/575).  I believe we're good.

`pip install -u apprise` and try again.
